packages/components/table/__tests__/table.test.ts

```diff
@@ -439,6 +441,82 @@ describe('Table.vue', () => {
     })
   })
 
+  describe('filter filter-icon slot', () => {
+    let wrapper: VueWrapper<ComponentPublicInstance>
+
+    beforeEach(async () => {
+      wrapper = mount({
+        components: {
+          ElTable,
+          ElTableColumn,
```

Comment:
```suggestion
          ElTableColumn,
          CaretBottom,
          CaretTop,
```

This can avoid using the `// eslint-disable-next-line @typescript-eslint/no-unused-vars`

Reply:
Looks great. 👍